patches/@ua+react-native-airship+15.3.1.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/node_modules/@ua/react-native-airship/react-native-airship.podspec b/node_modules/@ua/react-native-airship/react-native-airship.podspec
+index 5e0ce2d..4456f61 100644
+--- a/node_modules/@ua/react-native-airship/react-native-airship.podspec
++++ b/node_modules/@ua/react-native-airship/react-native-airship.podspec
+@@ -20,18 +20,7 @@ Pod::Spec.new do |s|
+ 
+   # Don't install the dependencies when we run `pod install` in the old architecture.
+   if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
+-    s.compiler_flags = folly_compiler_flags + " -DRCT_NEW_ARCH_ENABLED=1"
+-    s.pod_target_xcconfig    = {
+-        "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost\"",
+-        "OTHER_CPLUSPLUSFLAGS" => "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1",
+-        "CLANG_CXX_LANGUAGE_STANDARD" => "c++17"
+-    }
+-    s.dependency "React-Codegen"
+-    s.dependency "React-RCTFabric"
+-    s.dependency "RCT-Folly"
+-    s.dependency "RCTRequired"
+-    s.dependency "RCTTypeSafety"
+-    s.dependency "ReactCommon/turbomodule/core"
++    install_modules_dependencies(s)
```

Comment:
this was merged upstream and released in `17.2.0`. Let's upgrade to `17.2.1` and remove the patch